subsys/ipc/ipc_service/backends/Kconfig.rpmsg

```diff
@@ -12,4 +12,13 @@ config IPC_SERVICE_BACKEND_RPMSG_WQ_STACK_SIZE
 	  prevent notifying service users about received data from the system
 	  work queue. Size is the same for all instances.
 
+config IPC_SERVICE_BACKEND_RPMSG_SHMEM_RESET
+	bool "Reset shared memory state"
+	default y
```

Comment:
uhm, this should not usually be required so let's stick to `n`. "broken" platforms can set this to `y` instead.

Reply:
I set it up as you suggested